config/initializers/rack-attack.rb

```diff
@@ -4,7 +4,7 @@ class Rack::Attack
   # Always allow requests from localhost
   # (blacklist & throttles are skipped)
   whitelist('allow from localhost') do |req|
-    # Requests are allowed if the return value is truthy
```

Comment:
It seems you removed a comment character in front of text that doesn't look like code. Please take a look & re-submit, thanks!